docs/docs/getting-started/quickstart.md

```diff
@@ -245,10 +263,32 @@ id,name,age,city,age_group
 4,Diana,31,Los Angeles,Middle
 ```
 
+## Step 7. Deploy to the cloud (Optional)
+
+Once you have run your pipeline locally, you can optionally deploy it to the cloud.
```

Comment:
```suggestion
Once you have run your pipeline locally, you can optionally deploy it.
```

If we're showing an OSS deployment in the below tab, we may want to forgo the word "cloud".

Reply:
Good call, I can use "production" instead.